src/render.rs

```diff
@@ -138,4 +138,31 @@ mod tests {
         #[allow(dead_code)]
         fn takes_dyn_render(_: &dyn super::RootRender) {}
     }
+
+    #[test]
+    fn render_bump_scoped_child() {
+        use crate::{builder::*, bumpalo::collections::String, Node, Render, RenderContext};
+
+        struct Child<'a> {
+            name: &'a str,
+        }
+
+        impl<'a> Render<'a> for Child<'a> {
+            fn render(&self, _cx: &mut RenderContext<'a>) -> Node<'a> {
+                text(self.name)
+            }
+        }
+
+        struct Parent;
+
+        impl<'a> Render<'a> for Parent {
+            fn render(&self, cx: &mut RenderContext<'a>) -> Node<'a> {
+                let child_name = String::from_str_in("child", cx.bump).into_bump_str();
+
+                div(&cx)
+                    .children([Child { name: child_name }.render(cx)])
+                    .finish()
+            }
+        }
+    }
```

Comment:
This should act as a smoke test for bump-scoped lifetimes in child components

Reply:
Sorry I should have been more clear. This does ensure that code with bump-scoped child components compiles, but I was hoping for a rendering test that ensures that we also get the expected physical dom. Should be fine to put it in the `render.rs` test file.

Reply:
Ah, ok. I'll get that taken care of sometime tonight.